opendbc/car/honda/values.py

```diff
@@ -351,8 +351,8 @@ class CAR(Platforms):
 
   # port extensions
   HONDA_CLARITY = HondaNidecPlatformConfig(
-    [HondaCarDocs("Honda Clarity 2018-21", min_steer_speed=12. * CV.MPH_TO_MS)],
-    CarSpecs(mass=1834, wheelbase=2.75, centerToFrontRatio=0.4, steerRatio=16.5),
+    [HondaCarDocs("Honda Clarity 2018-2021", "All", min_steer_speed=3. * CV.MPH_TO_MS)],
+    CarSpecs(mass=4052. * CV.LB_TO_KG, wheelbase=2.75, centerToFrontRatio=0.4, steerRatio=16.50),
```

Comment:
```suggestion
    CarSpecs(mass=1838, wheelbase=2.75, centerToFrontRatio=0.4, steerRatio=16.5),
```
 
- `mass`: 1838 kg = 4052 lb. No functional benefit, using KG in mass is preferred.
- `centerToFrontRatio`: a 0.01 difference has negligible performance impact, but we can merge this if this in practice significant changes performance.
- `steerRatio`: 16.5 = 16.50. No functional benefit.

Reply:
Feedback implemented